js/game.js

```diff
@@ -105,25 +119,36 @@ function startSubgame(subgame) {
 };
 
 function betCheck() {
-    game.playerBet = game.rivalBet;
-    game.pendingBets[game.subgame] = game.rivalBet;
+    game.lastAction = LastAction.USER_CHECK;
+    game.lastBetStanding = game.rivalBet;
+    // needed? game.playerBet = game.lastBetStanding;
+    game.pendingBets[game.subgame] = game.lastBet;
```

Comment:
After trying to pass the tests, the lacking of the first line breaks one of them. We should either remove the test or readd the line and it's simpler for me to suggest the former from here.

The second line includes a fix caught by the tests.

```suggestion
    game.playerBet = game.lastBetStanding;
    game.pendingBets[game.subgame] = game.lastBetStanding;
```